### ISSUE TITLE
(more) translations needed

### ISSUE THREAD

**Issue by oberon-manjaro** [Chrysostomus/aif-dev#20]
Some common languages are still missing completely
**German** for example! ;)
Also there will be additional strings to translate as we procede. I don't see a proper translating system on the horizon, but for now I've just started marking new strings with ``# translate me!`` so we can at least easily search for them whenever we need the untranslated bits.

**Comment by Chrysostomus** [comment]
Excellent! Good thing we have many german speaking users.

**Comment by oberon-manjaro** [comment]
https://github.com/Chrysostomus/aif-dev/commit/0f2c83d7620a21e38702207b54c1964e96134ccb

**Comment by oberon-manjaro** [comment]
@tids I you have some time to spare maybe :wink: ... https://github.com/Chrysostomus/aif-dev/blob/master/translations/german.trans

**Comment by oberon-manjaro** [comment]
@tids , just you know: @eugen-b has already translated the btrfs section. Please don't do that one in any case :wink: We still need to decide if we use polite form or not. Personally I'm in favor of that. I think it makes a more professional impression, maybe - but then on the other hand it's also nice if the user feels more welcome and at home if we say you from the beginning, if you see what I mean ... :laughing: - (languages are fun, aren't they!?)

**Comment by Chrysostomus** [comment]
I think it also depends on the specific language/culture. In if you use the polite form in ui in finnish, you just sound like you desperately want to sell something. In german and russian I think I would use Sie/Вы. In english... I don't know.

**Comment by Chrysostomus** [comment]
Doing a finnish translation would be pointless though, because if a finnish person cannot understand this much english, they probably aren't appropriately equipped to use computers/linux in general.

**Comment by oberon-manjaro** [comment]
> In english...

you use you! :laughing: ,

while with Swedish you break Swedish law if you use the polite form :policewoman: 
German is just special... ... :smiling_imp:

**Comment by oberon-manjaro** [comment]
Eugen has assigned himself to translate the rest of the German file. Thank you for that! He'll do it in a "mildly polite" form ... :+1:

**Comment by oberon-manjaro** [comment]
I've prepared the German menu entry (commented out until ready) and made the menu use the column feature we just discovered. This will make adding more languages easier.
Naming and sorting of languages was inconsistent, and I suggest here the result of a longer recent discussion with @Huluti about the same problem in **manjaro-hello**. Result was that if you want to sort them alphabetically more or less the only way is sorting them by their English names keeping English the default (dialog has a feature for that, too, of course! :laughing: - dialog is just awesome!)

**Comment by oberon-manjaro** [comment]
https://github.com/Chrysostomus/aif-dev/commit/dae6716aad82f193d5cb6d6bb630a3bfc313c058

![](http://imgur.com/D8FIP51.png)

**Comment by fhdk** [comment]
Does the Danish translation need attention?

**Comment by oberon-manjaro** [comment]
Yes please! :-) look for the lines marked with "translate me"! :-)

**Comment by fhdk** [comment]
Done https://github.com/Chrysostomus/manjaro-architect/commit/eb4d577a78b8bfc7c6453b2962249566be8bcb7f
